Misc/NEWS.d/next/Library/2017-08-12-13-00-21.bpo-5001.tgVI5U.rst

```diff
@@ -0,0 +1,7 @@
+This is a fix of some of the uninformative asserts in `multiprocessing`,
```

Comment:
Well, it's now 1 short NEWS entry and I fixed most of the asserts. (Of course, then I had a typo due to an editor save problem... sigh. Unfortunately and ironically, the multiprocessing tests overload my current machine. Everything else works...)